### ISSUE TITLE
[Color Picker, Color Hex Codes, and Color Database] Color Picker, Color Hex Codes, and Color Database

### ISSUE THREAD

**Issue by BambuLabDiscord** [bambulab/BambuStudio#4471]
One of your biggest selling points is multicolor; yet, I have to keep entering RGB values... Please add a Color Picker, Color Hex Codes, and a Color Database for saving different filament color properties! 

A great example of a Color Picker and Color Hex Codes can be found in Fusion 360. I also Tweeted an image: fatboy1271

**Comment by fatboy1271** [comment]
![image](https://github.com/bambulab/BambuStudio/assets/77463808/b877276f-a086-4be4-ab89-c2fb89db6857)

**Comment by indyjason79** [comment]
Yes, we need this very much please! I'm getting tired of having to convert RGB/hex.

**Comment by InterruptionF10** [comment]
Agreed! Just bought 8 new Bambu Labs colors from their recent sale. Now having to convert their Hex code values to RBG for a new project I am working on. I want my assembly in the .3mf to closely match the colors I am going to see when I print and post it.
